ast/desugar/test/desugar_test.cc

```diff
@@ -30,3 +30,14 @@ TEST(DesugarTest, SimpleDesugar) { // NOLINT
     sorbet::core::MutableContext ctx(gs, sorbet::core::Symbols::root());
     auto o1 = sorbet::ast::desugar::node2Tree(ctx, move(ast));
 }
+
+TEST(DesugarTest, EmptyTreeDesugar) { // NOLINT
+    sorbet::core::GlobalState gs(errorQueue);
+    gs.initEmpty();
+    sorbet::core::UnfreezeNameTable nameTableAccess(gs);
+    sorbet::core::UnfreezeFileTable ft(gs);
+
+    auto ast = sorbet::parser::Parser::run(gs, "<test>", "\"#{}:\"");
+    sorbet::core::MutableContext ctx(gs, sorbet::core::Symbols::root());
+    auto o1 = sorbet::ast::desugar::node2Tree(ctx, move(ast));
+}
```

Comment:
This test case is redundant with the `test/testdata` test, and the other one is shorter and easier to grep for.

Do you mind deleting this?